source/auth/auth.md

```diff
@@ -1260,7 +1260,7 @@ in the MONGODB-OIDC specification, including sections or blocks that specificall
     performed after SRV record resolution, if applicable. This property is only required for drivers that support the
     [Human Authentication Flow](#human-authentication-flow).
 
-<div id="built-in-provider-integrations">
+<span id="built-in-provider-integrations">
```

Comment:
I'm not familiar with how Markdown handles HTML tags, but shouldn't these have a closing `</span>`?

Reply:
Updated